### DISCUSSION TITLE
How to determine if a string contains a substring? (in a template)

### DISCUSSION THREAD

**Discussion by jamietanna** [goreleaser/goreleaser#4548] [Q&A] [answered]
I'm using a mix of [go-semantic-release](https://github.com/go-semantic-release/semantic-release) and goreleaser to ship a CLI.

As part of this, I will expect my release notes to include specific text to indicate that there are breaking changes (as can be seen in https://gitlab.com/tanna.dev/dependency-management-data/-/releases/v0.71.0)

I'm looking to indicate within the [Slack announcement](https://goreleaser.com/customization/announce/slack/) as to whether there's any breaking changes by looking up the string in `.ReleaseNotes`

Is it possible to do this?

I.e. in Hugo I'd use https://gohugo.io/functions/strings/contains/ but I can't see a similar function

**Comment by caarlos0** [comment, marked as answer]
just added it in 5d44ed14fda18831c628109ad297502dfceabb1b 

will be in 1.24 and on next nightly builds


Thanks for the suggestion :)

**Reply by jamietanna** [reply]
Amazing, thanks so much :rocket:
